configs/forseti_conf.yaml.sample

```diff
@@ -110,6 +110,8 @@ scanner:
           enabled: true
         - name: iam_policy
           enabled: true
+        - name: forwarding_rules
```

Comment:
nit: the other scanner names are singular, so can we can be consistent here too?